packages/workspace/src/workspace/workspace.ts

```diff
@@ -758,7 +759,8 @@ export const loadWorkspace = async (
         stateToBuild.states[envName].mapVersions,
         stateToBuild.states[envName].merged,
         changeResult.cacheValid,
-        element => getCustomReferences(element, currentEnvConf().accountToServiceName ?? {}),
+        element => getCustomReferences(element, currentEnvConf().accountToServiceName ?? {}, adaptersConfig),
```

Comment:
nit, because this is confusing when it is actually an array
```suggestion
        elements => getCustomReferences(elements, currentEnvConf().accountToServiceName ?? {}, adaptersConfig),
```